lisa/util/constants.py

```diff
@@ -114,3 +114,8 @@
 TESTCASE_RETRY = "retry"
 TESTCASE_USE_NEW_ENVIRONMENT = "use_new_environment"
 TESTCASE_IGNORE_FAILURE = "ignore_failure"
+
+# data disk caching type
+DATADISK_CACHING_TYPE_NONE = "None"
```

Comment:
It might be better to add it as an enum similar `DiskType` in disks.py.

Reply:
I encounter circular import issue, when use the class in the schema.py